### ISSUE TITLE
Prep for Spanish

### ISSUE THREAD

**Issue by jewkesy** [jewkesy/alexa-popcorn#39]

**Comment by jewkesy** [comment]
Submitted for certification

**Comment by jewkesy** [comment]
1. The example phrases that you chose to present to users in the companion app currently use unsupported launch phrasing.

Actual:"Alexa, preguntar a Popcorn Quiz por mi puntaje" 
Expected: "Alexa, pregúntale a Popcorn Quiz por mi puntaje" 

Third example phrase:

Actual: "Alexa, preguntar a Popcorn Quiz por mi rango"
Expected:"Alexa, pregúntale a Popcorn Quiz por mi rango" 

Please see our documentation on understanding how users invoke custom skills for more information on supported launch phrasing.


2. Images in home cards must display without error. We found home card(s) with images that were not displayed properly.

Steps To Reproduce:

User: "Alexa, jugar Popcorn Quiz"
Skill: "Cierto o falso: ¿El viaje, lanzado en 2016, tiene los siguientes personajes: Tony Blair, Jack, Martin McGuinness y Ian Paisley?"

The image of the home card is missing. 

Please refer to test case 3.3 of the Submission Checklist for expected behavior.


3. The skill does not exit/respond appropriately when users say “stop” or “cancel”.

Steps To Reproduce:

User:"Alexa, jugar Popcorn Quiz"
Skill: "Cierto o falso: ¿El viaje, lanzado en 2016, tiene los siguientes personajes: Tony Blair, Jack, Martin McGuinness y Ian Paisley?"
User: "Alexa, stop"
Skill: "Incorrecto. La respuesta correcta es Mike Lazarus, Claire Lochewood, Angela O'Murphy y Eryn Lazarus. Próxima pregunta. ¿Delirium, lanzado en 2018, caracteristicas Griffin Freeman, Ryan Pinkston, Mike C. Manning y August Roads?"

Please note that the command "stop" keeps the session open instead of closing it and the answer is understood by the skill as "true" or "false".

Please see test case 4.13 from our Submission Checklist for guidance on skill exiting.


4. When attempting to invoke the skill using an example present in the example phrases or the skill’s description, the skill returns a non- contextualized response or error. All customer facing example phrases must function without error since these are interactions that users are most likely to try.

Issue:  TTS response

Steps to Reproduce:

User: "Alexa, jugar popcorn quiz"
Skill: "Cinco preguntas por venir. Si no sabes, adivina. Primera pregunta. Cierto o falso, ¿Dead Still, lanzado en 2014, caracteristicas Sam Soghor, Tashiana Washington, Meeko y Ty Hickson?"

Please note that the skill reads the question marks at the beginning of each question of the quiz. You should correct that in order to have a better understanding of the question. 


5. The description displayed in the skill’s detail card must contain accurate information so users are well informed on how to use the skill.

Please change the next sentences in the long description as following:

Actual: "Cuantos más juegos juegues y más preguntas respondas correctamente, mayores serán las posibilidades de aumentar tu rango mundial."
Expected: "Cuantos más juegos practicas y más preguntas respondas correctamente, mayores serán las posibilidades de aumentar tu rango mundial."

Please see test case 3.2 from our Submission Checklist for guidance on skill descriptions.

**Comment by jewkesy** [comment]
1) Done

**Comment by jewkesy** [comment]
5. Done

**Comment by jewkesy** [comment]
2. Could not replicate - image exists

https://images-na.ssl-images-amazon.com/images/M/MV5BMTQxMzNlZGQtYTU0My00NzFmLWE4NmYtODc4Y2Q0OTY2ODk2XkEyXkFqcGdeQXVyMTc4NDkxOA@@._V1_SX300.jpg

**Comment by jewkesy** [comment]
4.  Posted on Reddit for assistance on sentence structure

https://www.reddit.com/r/Spanishhelp/comments/909m6g/can_someone_help_me_with_the_translation_of_a/

**Comment by jewkesy** [comment]
Cierto o falso, ¿Dead Still, lanzado en 2014, caracteristicas Sam Soghor, Tashiana Washington, Meeko y Ty Hickson?

¿Cierto o falso? Dead Still, lanzado en 2014, es una película protagonizada por Sam Soghor, Tashiana Washington, Meeko y Ty Hickson

**Comment by jewkesy** [comment]
Back in certification!

**Comment by jewkesy** [comment]
"Alexa, juega popcorn quiz"

"Vale, hay cinco preguntas. Si no sabes la repuesta, adivínala. Primera pregunta: ¿Verdadero o falso? La película Dead Still, que se lanzó en 2014, es protagonizada por Sam Soghoe, Tashiana Washington, Meeko, y Tu Hickson.

This is the best way to put everything

Edit: Autocorrect changed one of the names

Edit 2: Since this is for Spain I changed "cierto" to "verdadero" because when I took classes in Spain true or falso questions were "verdadero o falso" questions not "cierto o falso"
